docs/source/reference/job-queue.rst

```diff
@@ -1,24 +1,25 @@
 .. _job-queue:
 
-Cluster Job Queue
-=================
+Cluster Jobs
```

Comment:
Can we at least have a section mentioning `sky launch` on existing cluster submits the job to the existing cluster but with additional steps, including `file_mounts` and `setup`?

Whether `sky launch` creates a new cluster is a common question we got. (people were afraid of doing `sky launch` on an existing cluster)

Reply:
Done.